src/g_input.c

```diff
@@ -46,7 +46,7 @@ void G_ClearLocalView(void)
     memset(&localview, 0, sizeof(localview));
 }
 
-void G_UpdateLocalView_FakeLongTics(void)
+static void UpdateLocalView_FakeLongTics(void)
```

Comment:
Removing `static` on this line fixes the latency problem😐

Reply:
`__declspec(noinline)` also works

Reply:
How is the mouse input when not using fake_longtics? Let's say with uncapped and raw_input enabled. Does your bisecting lead back to this commit as well?

Reply:
raw_input when not using fake_longtics has always been good, it doesn't seem to be affected by any of this.